tests/Integration/tests/body-elements/line-blocks/expected/index.html

```diff
@@ -1,38 +1,17 @@
-
 <!-- content start -->
     <div class="section" id="line-blocks">
     <h1>Line Blocks</h1>
 
-        <div class="line-block">
-            <div class="line">
-                Line blocks are useful for addresses,
-            </div>
-            <div class="line">
-                verse, and adornment-free lists.
-            </div>
-            <div class="line">
-                <br>
-            </div>
-            <div class="line">
-                Each new line begins with a
-            </div>
-            <div class="line">
-                vertical bar (“|”).
-            </div>
-            <div class="line-block">
-                <div class="line">
-                    Line breaks and initial indents
-                </div>
-                <div class="line">
-                    are preserved.
-                </div>
-            </div>
-            <div class="line">
-                Continuation lines are wrapped
-                portions of long lines; they begin
-                with spaces in place of vertical bars.
-            </div>
-        </div>
+            <p>| Line blocks are useful for addresses,
```

Comment:
Shouldn't this remain using `<div>` output style tags?

Reply:
arg true. Copying real output into expected should exclude unrelated skipped tests